tests/system/_sample_data.py

```diff
@@ -90,5 +90,8 @@ def _check_cell_data(found_cell, expected_cell, recurse_into_lists=True):
         for found_item, expected_item in zip(found_cell, expected_cell):
             _check_cell_data(found_item, expected_item)
 
+    elif isinstance(found_cell, float):
+        assert abs(found_cell - expected_cell) < 0.00001
```

Comment:
Similar to JAVA: https://github.com/googleapis/java-spanner/blob/main/google-cloud-spanner/src/test/java/com/google/cloud/spanner/it/ITFloat32Test.java#L357C42-L357C51